addon/pages/train.html

```diff
@@ -99,25 +95,27 @@ <h1>Trainer</h1>
       <div>
       </div>
       <div>
-        <meter id="progress" style="display: none"></meter>
+        <meter id="progress" class="hidden"></meter>
       </div>
       <div>
       </div>
       <div id="caveats" class="bottommost">
         Standardized window size will be used for consistent sizing of elements. Training is stochastic, so different runs may reach different results.
       </div>
-      <div class="output label">
-        Coefficients:
-      </div>
-      <div id="coeffs"></div>
-      <div class="output label">
-        Accuracy:
-      </div>
-      <div id="accuracy"></div>
-      <div class="output label">
-        Good/Bad:
+      <div id="output" class="hidden uiGrid">
+          <div class="label">
```

Comment:
I thought for a moment you had improved my markup and added `<label>` elements. :-) Don't take any of my code as sacred; this is "research quality" right now, doubly quick because I was rushing to make a quarterly goal.